GiEnJul/Repositories/RecipientRepository.cs

```diff
@@ -44,6 +45,16 @@ public async Task<List<Models.Recipient>> GetUnmatchedRecipientsAsync(string loc
             return _mapper.Map<List<Models.Recipient>>(recipients);
         }
 
+        public async Task<bool> RecipientDoesExist(string referenceId)
+        {
+            var query = $"ReferenceId eq '{referenceId}";
+            var exists = await GetAllByQueryAsync(query);
+            if (_mapper.Map<List<Models.Recipient>>(exists).Count > 0) return true;
```

Comment:
```suggestion
            if (_mapper.Map<List<Models.Recipient>>(exists).Any()) 
                 return true;
```
But I think this change also requires "using linq" 🤷‍♂️
Anyways, please move the return down

Reply:
fixed